package.json

```diff
@@ -19,9 +19,17 @@
     "type": "git",
     "url": "git://github.com/PolymerElements/marked-element.git"
   },
-  "license": "https://polymer.github.io/LICENSE.txt",
+  "license": "BSD-3-Clause",
```

Comment:
I dont think we can make this change here, we would need to update every repository at once if we need to.

Reply:
Okay, may be worth doing since NPM throws a warning about non-SPDX license. I can revert this if y'all would like.